build/lib/backend_http/do_request.py

```diff
@@ -0,0 +1,36 @@
+from urllib.parse import urlparse
+
+import requests
+
+
+class InvalidUrl(Exception):
+    pass
+
+
+class HttpVerbNotImplemented(Exception):
+    pass
+
+
+def do_request(http_verb, url):
```

Comment:
No creo que esta indirección sea necesaria. Deberías poder importar `get` y listo.

Reply:
Not sure if I quite catch your point, could you elaborate more on this?
Anyways 🤔 `do_request` is not doing the same that `get` does. In that case, I'd prefer to maintain both with their corresponding responsibilities.